pinc/wordcheck_engine.inc

```diff
@@ -707,6 +707,19 @@ function count_wordcheck_suggestion_events($projectid, $start_time = 0)
     return $row["numevents"];
 }
 
+/**
+  * @ param string sep
+  *     The separator between substrings
+  * @ param string $string
+  *     The input string to split
+  *
+  * @ return Array<string>
+  */
+function _explode_no_empty($sep, $string)
```

Comment:
According to the DP Code best practices, the function declaration would be after the use.

Reply:
Done.